applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -633,7 +633,7 @@ int le_audio_enable(le_audio_receive_cb recv_cb, le_audio_timestamp_cb timestmp_
 
 	LOG_INF("Scanning for broadcast sources");
 
-	ret = bt_audio_broadcast_sink_scan_start(BT_LE_SCAN_PASSIVE);
+	ret = bt_audio_broadcast_sink_scan_start(BT_LE_SCAN_PARAMS);
```

Comment:
```suggestion
	ret = bt_audio_broadcast_sink_scan_start(NRF5340_AUDIO_GATEWAY_SCAN_PARAMS);
```

Reply:
It was failing compile because of this

Reply:
Thanks. Missed that one. Fixed in the last force-push/squash action. Let's see if there's still compliance issues.